Tools/Pipeline/Windows/ContentIcons.cs

```diff
@@ -25,6 +25,7 @@ public class ContentIcons: IDisposable
         public ContentIcons()
         {
             Icons = new ImageList();
+            Icons.ColorDepth = ColorDepth.Depth32Bit;
```

Comment:
Very weird... but if it works then it works.  I don't see any potential side effects from this.